common-lib/src/main/scala/com/gu/mediaservice/lib/auth/Store.scala

```diff
@@ -66,6 +66,7 @@ class KeyStore(bucket: String, credentials: AWSCredentials) extends BaseStore[St
 object PermissionType extends Enumeration {
```

Comment:
I also would tend to prefer case objects and traits, we could change this in a future PR if nobody has a particular preference for `Enumeration`.